apm-agent-core/src/main/java/co/elastic/apm/agent/report/IntakeV2ReportingEventHandler.java

```diff
@@ -56,7 +57,7 @@ public void init(ApmServerReporter reporter) {
     }
 
     @Override
-    public void onEvent(ReportingEvent event, long sequence, boolean endOfBatch) {
+    public void onEvent(ReportingEvent event, long sequence, boolean endOfBatch) throws Exception {
```

Comment:
What happens when we propagate the exception to the disruptor? Which exception handler do we have configured? Is there any chance of this preventing this event handler to process further events?

Reply:
It doesn't seem we configured one. We should, otherwise the default handler throws an exception as well.
However, there is nothing special here, any NPE or other `RuntimeException` caused by one of the many things we try to do within this loop could fail and cause event processing halt.

Reply:
The usage of JUL is fine as we replace that with our JulBridgeLogger.

This is a snippet of the decompiled class that we ship:

```java
import co.elastic.apm.agent.logging.JulBridgeLogger;
import java.util.logging.Level;

public final class IgnoreExceptionHandler implements ExceptionHandler<Object> {
    private static final JulBridgeLogger LOGGER = JulBridgeLogger.getLogger(IgnoreExceptionHandler.class.getName());
    private final JulBridgeLogger logger;
```

Reply:
Right! I completely forgot about `JulBridgeLogger` even though I just recently refactored it 🤦 
We're all good then.